index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+
+<body>
+   
+    <div class="flex-container">
+    <header>
+        <nav class="navbar">
+            <ul>
+                <li class="nav-logo">My Logo</li>
+                
+            </ul>
+        </nav>
+    </header>
+    <div class="top-images">
+        <img src="images/Ellipse3.png" alt="ellipse" class="ellipse-3">
+        <img src="images/Rectangle4.png" alt="ellipse" class="rectangle-4">
+        <img src="images/Ellipse11.png" alt="ellipse" class="ellipse-11">
+    </div>
+   
+    <section class="greet-section"> 
+ 
+        <h2>I’m Mariane Glad to see you!</h2>
+    
+        <p class="info-section">I’m a software developer! I can help you build a product , 
+        feature or website Look through some of my work and experience! 
+        If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    
+        <p class="connect">LET’S CONNECT</p>
+    </section>
+    <div class="connect-links">
+        <img src="images/Vectortweet.svg" alt="The twitter logo" class="twitter">
+        <img src="images/Vectori.png" alt="Linked in logo" class="linkedin">
+        <img src="images/Vectorn.png" alt="Linked in logo" class="linkedin">
+        <img src="images/Vectorm.svg" alt="M logo" class="mlogo">
+        <img src="images/Vectorgit.svg" alt="Github logo" class="github">
+        <img src="images/Vectorpeace.svg" alt="Peace Sign" class="peace-sign">
+    </div>
```

Comment:
- [ ] These should be icons, not images. You can use font awesome for the purpose. 
- [ ] The list of icons should use a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag)